### PR TITLE
MM- 18893: Change text User to Member - Self-Managed

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-33106

Updated:
- Self-Managed Admin Guide > Onboard Users > Guest Accounts (Beta) (E10/E20) > Promoting and Demoting User Roles
   - Added additional context around how to promote and demote user accounts from guests to members and vice versa

### CODE REVIEW

File: source/deployment/guest-accounts.rst [hunk 1]
```diff
@@ -27,12 +27,14 @@ Additionally, guests are not automatically added to the default ``Town-square``
...
-4. (Optional) **Enforce Multi-Factor Authentication (MFA) for your Guests**.
- - If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users.
+1. Navigate to **System Console > Authentication > Guest Access (Beta)**.
+2. Set **Enable Guest Access** to **true**.
+3. (Optional) Specify the **Whitelisted Guest Domains** that are acceptable for guest access. This allows the System Admin to set a list of approved guest domains. 
+4. (Optional) Set **Enforce Multi-factor Authentication** to **true**. If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users.
+ 
+.. note::
+
+  If you have team domain restrictions, you also need to add your guest domain by going to **System Console > User Management > Teams**, selecting a team, then enabling **Only specific email domains can join this team**.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:37]:
```suggestion
  If you have team domain restrictions, you also need to add your guest domain in **System Console > User Management > Teams**. Select a team, then enable **Only specific email domains can join this team**.
```

File: source/deployment/guest-accounts.rst [hunk 2]
```diff
@@ -69,7 +72,7 @@ When enabled, the **Guest Filter** in Mattermost identifies external users whose
 1. Enable Guest Access via **System Console > Authentication > Guest Access (Beta)**.
 2. Navigate to **System Console > Authentication > AD/LDAP**.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:73]:
```suggestion
2. Go to **System Console > Authentication > AD/LDAP**.
```

File: source/deployment/guest-accounts.rst [hunk 3]
```diff
@@ -85,14 +88,14 @@ If a Mattermost guest user has the guest role removed in the SAML system, the sy
 1. Enable Guest Access via **System Console > Guest Access (Beta)**.
 2. Navigate to **System Console > Authentication > SAML 2.0**.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:89]:
```suggestion
2. Go to **System Console > Authentication > SAML 2.0**.
```

File: source/deployment/guest-accounts.rst [hunk 4]
```diff
@@ -85,14 +88,14 @@ If a Mattermost guest user has the guest role removed in the SAML system, the sy
...
 3. Complete the **Guest Attribute** field.
-4. Choose **Save**.
+4. Select **Save**.
 
 When a guest logs in without having any channels assigned to their account, they're advised to contact an administrator.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:93]:
```suggestion
When a guest logs in without having any channels assigned to their account, they're advised to contact a System Admin.
```

File: source/deployment/guest-accounts.rst [hunk 5]
```diff
@@ -85,14 +88,14 @@ If a Mattermost guest user has the guest role removed in the SAML system, the sy
...
 
 Guest Permission Settings
 -------------------------
 
-In E10 and E20, you can also control which users can invite guests. By default, only the System Admins can invite guests. There are `additional permissions <https://docs.mattermost.com/deployment/advanced-permissions.html>`_ in E20 that can be adjusted under **System Console > Advanced Permissions > System Scheme** to control a guest’s ability to:
+In E10 and E20, you can also control which users can invite guests. By default, only the System Admins can invite guests. There are `additional permissions <https://docs.mattermost.com/deployment/advanced-permissions.html>`_ in E20 that can be adjusted under **System Console > User Management > Permissions > System Scheme** to control a guest’s ability to:
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:98]:
```suggestion
In Enterprise Edition E10 and E20, you can also control which users can invite guests. By default, only the System Admins can invite guests. There are `additional permissions <https://docs.mattermost.com/deployment/advanced-permissions.html>`_ in E20 that can be adjusted under **System Console > User Management > Permissions > System Scheme** to control a guest’s ability to:
```

File: source/deployment/guest-accounts.rst [hunk 6]
```diff
@@ -126,25 +129,24 @@ When a guest has been removed from all channels within a team, and if they belon
 Promoting and Demoting User Roles
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-System Admins can demote a member to guest by updating their role in **System Console > User Management > Users**. The demoted user will retain their existing channel and team memberships but will be restricted from discovering public channels and collaborating with users outside of the channels they are in.  This is useful if you are already collaborating with external contractors and want to restrict their abilities within your Mattermost instance.
+System Admins can demote a member to a guest by updating their role in **System Console > User Management > Users**. Select the member, then select **Demote to Guest**. The demoted user retains their existing channel and team memberships but will be restricted from discovering public channels and collaborating with users outside of the channels they are in. This is useful if you are already collaborating with external contractors, and want to restrict their abilities within your Mattermost instance.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:132]:
```suggestion
System Admins can demote a member to a guest by updating their role in **System Console > User Management > Users**. Select the member, then select **Demote to Guest**. 

The demoted user retains their existing channel and team memberships but is restricted from discovering public channels and collaborating with users outside of the channels they're in. This is useful if you're already collaborating with external contractors, and want to restrict their abilities within Mattermost.
```

File: source/deployment/guest-accounts.rst [hunk 7]
```diff
@@ -126,25 +129,24 @@ When a guest has been removed from all channels within a team, and if they belon
...
 
-System Admins can also promote a guest to member by updating their role in **System Console > User Management > Users**.
+System Admins can also promote a guest to member by updating their role in **System Console > User Management > Users**. Select the guest, then select **Promote to Member**
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:134]:
```suggestion
System Admins can also promote a guest to member by updating their role in **System Console > User Management > Users**. Select the guest, then select **Promote to Member**.
```

File: source/deployment/guest-accounts.rst [hunk 8]
```diff
@@ -126,25 +129,24 @@ When a guest has been removed from all channels within a team, and if they belon
...
+To disable the Guest Accounts feature, go to **System Console > Authentication > Guest Access (Beta)**, then set **Enable Guest Access** to **False**. To deactivate individual guest accounts, go to **System Console > User Management > Users**. Select a user, then select **Deactivate**. You can re-activate individual guest accounts by selecting **Activate**.
 
-From 5.18, if you're using AD/LDAP and the Guest Access setting is disabled, the Guest Filter and existing guest users in System Console are deactivated. Additionally, no new guests can be invited or added using the filter as an authentication method. If a previous guest's credentials match the user filter (the only filter which is active when Guest Access is disabled), they will be reactivated and promoted to a member user upon their next login.
+From version 5.18, when a single Guest Account is deactivated or the Guest Account feature is disabled, guests are marked as ``inactive``, guests are logged out of Mattermost, and all guest sessions are revoked. In versions prior to 5.18, disabling the Guest Account feature leaves current Guest Accounts as active until they are manually deactivated.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:144]:
```suggestion
From Mattermost Server version 5.18, when a single Guest Account is deactivated or the Guest Account feature is disabled, guests are marked as ``inactive``, are logged out of Mattermost, and all guest sessions are revoked. In Mattermost Server versions prior to 5.18, disabling the Guest Account feature leaves current Guest Accounts as active until they are manually deactivated.
```

File: source/deployment/guest-accounts.rst [hunk 9]
```diff
@@ -126,25 +129,24 @@ When a guest has been removed from all channels within a team, and if they belon
...
 
-To disable the Guest Accounts feature, go to **System Console > Authentication > Guest Access (Beta) > Enable Guest Access** and select ``False``. In versions prior to 5.18, current Guest Accounts will remain active until guest users are manually marked ``inactive`` in **System Console > User Management > Users**.
+From 5.18, if you're using AD/LDAP and the Guest Access setting is disabled, the Guest Filter and existing guest users in System Console are deactivated. Additionally, no new guests can be invited or added using the filter as an authentication method. If a previous guest's credentials match the user filter (the only filter which is active when Guest Access is disabled), they will be reactivated and promoted to a member user upon their next login.
```
Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:146]:
```suggestion
From Mattermost Server version 5.18, if you're using AD/LDAP and the Guest Access setting is disabled, and the Guest Filter and existing guest users in System Console are deactivated. Additionally, no new guests can be invited or added using the filter as an authentication method. If a previous guest's credentials match the user filter (the only filter which is active when Guest Access is disabled), they will be reactivated and promoted to a member user upon their next login.
```

**Review by justinegeffen — COMMENTED** [review]
Great update! I noticed some legacy instructions ("Navigate to" as opposed to "Go to") which I corrected. Some other minor rewording suggestions but otherwise looks good!

**Comment by cwarnermm** [comment]
Thanks for the great review, @justinegeffen!

**Review by sbishel — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]
Thanks, @cwarnermm! There are three minor outstanding comments, but otherwise it looks good to me. :)

**Review by adamjclarkson — APPROVED** [review]

**Pull request merged**